Makefile

```diff
@@ -0,0 +1,6 @@
+check:
+	flake8 --config .flake8-code .
+	pylint lookout
```

Comment:
Why do we need pylint? It is noisy as hell and much harder to maintain for a large team compared to flake8.

Reply:
I would rather exclude it as in my experience it brings too much overhead for little benefits.

Reply:
so here I use it for only a few select errors (as can be seen in .pylintrc I exclude all, and then add just a few). It's just that pylint's docparams plugin seems to be the best option available to detect missing/faulty params in docstrings.